weblate/static/editor/full.js

```diff
@@ -210,6 +217,19 @@
     });
   };
 
+  FullEditor.prototype.removeTranslationEntry = function (delete_url) {
+    $.ajax({
+      type: "DELETE",
+      url: delete_url,
+      success: () => {
+        addAlert("Successfully Removed!");
```

Comment:
Any user displayed messages should be localizable:

```suggestion
        addAlert(getttext("Translation memory entry removed."));
```